From: Pietro Battiston <me@pietrobattiston.it>
Date: Wed, 21 Dec 2011 16:24:15 +0100
Subject: Fix test for Python 2.7

Imported 07f651929f025a0cab7020038e2ff235c85804c7 from upstream, to cope with
new floats representation.

Forwarded: not-needed
Author: Sean Gillies <sean.gillies@gmail.com>

---
 shapely/tests/MultiPolygon.txt |   16 ++++++++--------
 shapely/tests/Point.txt        |    4 ++--
 shapely/tests/Polygon.txt      |    6 +++---
 3 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/shapely/tests/MultiPolygon.txt b/shapely/tests/MultiPolygon.txt
index 8b8723a..376ddb6 100644
--- a/shapely/tests/MultiPolygon.txt
+++ b/shapely/tests/MultiPolygon.txt
@@ -9,23 +9,23 @@ Initialization
 From coordinate tuples
 
   >>> geom = MultiPolygon( [
-  ... (((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (1.0, 0.0)), [((0.1,0.1), (0.1,0.2), (0.2,0.2), (0.2,0.1))])
+  ... (((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (1.0, 0.0)), [((0.25,0.25), (0.25,0.5), (0.5,0.5), (0.5,0.25))])
   ... ] )
   >>> geom # doctest: +ELLIPSIS
   <shapely.geometry.multipolygon.MultiPolygon object at ...>
   >>> len(geom.geoms)
   1
   >>> geom.wkt
-  'MULTIPOLYGON (((0.0000000000000000 0.0000000000000000, 0.0000000000000000 1.0000000000000000, 1.0000000000000000 1.0000000000000000, 1.0000000000000000 0.0000000000000000, 0.0000000000000000 0.0000000000000000), (0.1000000000000000 0.1000000000000000, 0.1000000000000000 0.2000000000000000, 0.2000000000000000 0.2000000000000000, 0.2000000000000000 0.1000000000000000, 0.1000000000000000 0.1000000000000000)))'
+  'MULTIPOLYGON (((0.0000000000000000 0.0000000000000000, 0.0000000000000000 1.0000000000000000, 1.0000000000000000 1.0000000000000000, 1.0000000000000000 0.0000000000000000, 0.0000000000000000 0.0000000000000000), (0.2500000000000000 0.2500000000000000, 0.2500000000000000 0.5000000000000000, 0.5000000000000000 0.5000000000000000, 0.5000000000000000 0.2500000000000000, 0.2500000000000000 0.2500000000000000)))'
 
 Or from polygons
 
-  >>> p = Polygon(((0, 0), (0, 1), (1, 1), (1, 0)), [((0.1,0.1), (0.1,0.2), (0.2,0.2), (0.2,0.1))])
+  >>> p = Polygon(((0, 0), (0, 1), (1, 1), (1, 0)), [((0.25,0.25), (0.25,0.5), (0.5,0.5), (0.5,0.25))])
   >>> geom = MultiPolygon([p])
   >>> len(geom.geoms)
   1
   >>> print geom.wkt
-  MULTIPOLYGON (((0.0000000000000000 0.0000000000000000, 0.0000000000000000 1.0000000000000000, 1.0000000000000000 1.0000000000000000, 1.0000000000000000 0.0000000000000000, 0.0000000000000000 0.0000000000000000), (0.1000000000000000 0.1000000000000000, 0.1000000000000000 0.2000000000000000, 0.2000000000000000 0.2000000000000000, 0.2000000000000000 0.1000000000000000, 0.1000000000000000 0.1000000000000000)))
+  MULTIPOLYGON (((0.0000000000000000 0.0000000000000000, 0.0000000000000000 1.0000000000000000, 1.0000000000000000 1.0000000000000000, 1.0000000000000000 0.0000000000000000, 0.0000000000000000 0.0000000000000000), (0.2500000000000000 0.2500000000000000, 0.2500000000000000 0.5000000000000000, 0.5000000000000000 0.5000000000000000, 0.5000000000000000 0.2500000000000000, 0.2500000000000000 0.2500000000000000)))
 
 Or from another multi-polygon
 
@@ -33,7 +33,7 @@ Or from another multi-polygon
   >>> len(geom2.geoms)
   1
   >>> print geom2.wkt
-  MULTIPOLYGON (((0.0000000000000000 0.0000000000000000, 0.0000000000000000 1.0000000000000000, 1.0000000000000000 1.0000000000000000, 1.0000000000000000 0.0000000000000000, 0.0000000000000000 0.0000000000000000), (0.1000000000000000 0.1000000000000000, 0.1000000000000000 0.2000000000000000, 0.2000000000000000 0.2000000000000000, 0.2000000000000000 0.1000000000000000, 0.1000000000000000 0.1000000000000000)))
+  MULTIPOLYGON (((0.0000000000000000 0.0000000000000000, 0.0000000000000000 1.0000000000000000, 1.0000000000000000 1.0000000000000000, 1.0000000000000000 0.0000000000000000, 0.0000000000000000 0.0000000000000000), (0.2500000000000000 0.2500000000000000, 0.2500000000000000 0.5000000000000000, 0.5000000000000000 0.5000000000000000, 0.5000000000000000 0.2500000000000000, 0.2500000000000000 0.2500000000000000)))
 
 Sub-geometry Access
 -------------------
@@ -41,7 +41,7 @@ Sub-geometry Access
   >>> geom.geoms[0] # doctest: +ELLIPSIS
   <shapely.geometry.polygon.Polygon object at ...>
   >>> geom.geoms[0].wkt
-  'POLYGON ((0.0000000000000000 0.0000000000000000, 0.0000000000000000 1.0000000000000000, 1.0000000000000000 1.0000000000000000, 1.0000000000000000 0.0000000000000000, 0.0000000000000000 0.0000000000000000), (0.1000000000000000 0.1000000000000000, 0.1000000000000000 0.2000000000000000, 0.2000000000000000 0.2000000000000000, 0.2000000000000000 0.1000000000000000, 0.1000000000000000 0.1000000000000000))'
+  'POLYGON ((0.0000000000000000 0.0000000000000000, 0.0000000000000000 1.0000000000000000, 1.0000000000000000 1.0000000000000000, 1.0000000000000000 0.0000000000000000, 0.0000000000000000 0.0000000000000000), (0.2500000000000000 0.2500000000000000, 0.2500000000000000 0.5000000000000000, 0.5000000000000000 0.5000000000000000, 0.5000000000000000 0.2500000000000000, 0.2500000000000000 0.2500000000000000))'
   >>> geom.geoms[1]
   Traceback (most recent call last):
   ...
@@ -51,14 +51,14 @@ Geo interface
 -------------
 
   >>> geom.__geo_interface__
-  {'type': 'MultiPolygon', 'coordinates': [[((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (1.0, 0.0), (0.0, 0.0)), ((0.10000000000000001, 0.10000000000000001), (0.10000000000000001, 0.20000000000000001), (0.20000000000000001, 0.20000000000000001), (0.20000000000000001, 0.10000000000000001), (0.10000000000000001, 0.10000000000000001))]]}
+  {'type': 'MultiPolygon', 'coordinates': [[((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (1.0, 0.0), (0.0, 0.0)), ((0.25, 0.25), (0.25, 0.5), (0.5, 0.5), (0.5, 0.25), (0.25, 0.25))]]}
 
 Adapter
 -------
 
   >>> from shapely.geometry import asMultiPolygon
   >>> coords = ((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (1.0, 0.0))
-  >>> holes_coords = [((0.1,0.1), (0.1,0.2), (0.2,0.2), (0.2,0.1))]
+  >>> holes_coords = [((0.25,0.25), (0.25,0.5), (0.5,0.5), (0.5,0.25))]
   >>> mpa = asMultiPolygon([(coords, holes_coords)])
   >>> len(mpa.geoms)
   1
diff --git a/shapely/tests/Point.txt b/shapely/tests/Point.txt
index 2325afe..7ef16cb 100644
--- a/shapely/tests/Point.txt
+++ b/shapely/tests/Point.txt
@@ -121,8 +121,8 @@ Adapter
   >>> coords[0] = 1.0
   >>> pa.wkt
   'POINT (1.0000000000000000 4.0000000000000000)'
-  >>> pa.distance(p)
-  4.1231056256176606
+  >>> '%.10f' % pa.distance(p)
+  '4.1231056256'
 
   Now, the inverse
   
diff --git a/shapely/tests/Polygon.txt b/shapely/tests/Polygon.txt
index 4a9e5dd..93721b5 100644
--- a/shapely/tests/Polygon.txt
+++ b/shapely/tests/Polygon.txt
@@ -77,7 +77,7 @@ Ring Access
 Interior rings (holes)
 ----------------------
 
-  >>> polygon = Polygon(coords, [((0.1,0.1), (0.1,0.2), (0.2,0.2), (0.2,0.1))])
+  >>> polygon = Polygon(coords, [((0.25,0.25), (0.25,0.5), (0.5,0.5), (0.5,0.25))])
   >>> len(polygon.exterior.coords)
   5
   >>> len(polygon.interiors[0].coords)
@@ -103,13 +103,13 @@ Geo interface
 -------------
 
   >>> polygon.__geo_interface__
-  {'type': 'Polygon', 'coordinates': (((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (2.0, -1.0), (0.0, 0.0)), ((0.10000000000000001, 0.10000000000000001), (0.10000000000000001, 0.20000000000000001), (0.20000000000000001, 0.20000000000000001), (0.20000000000000001, 0.10000000000000001), (0.10000000000000001, 0.10000000000000001)))}
+  {'type': 'Polygon', 'coordinates': (((0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (2.0, -1.0), (0.0, 0.0)), ((0.25, 0.25), (0.25, 0.5), (0.5, 0.5), (0.5, 0.25), (0.25, 0.25)))}
 
 
 Adapter
 -------
 
-  >>> hole_coords = [((0.1,0.1), (0.1,0.2), (0.2,0.2), (0.2,0.1))]
+  >>> hole_coords = [((0.25,0.25), (0.25,0.5), (0.5,0.5), (0.5,0.25))]
   >>> from shapely.geometry import asPolygon
   >>> pa = asPolygon(coords, hole_coords)
   >>> len(pa.exterior.coords)
-- 
